test/integration_test.py

```diff
@@ -158,15 +164,66 @@ class IntegrationTestCase(AzulTestCase, metaclass=ABCMeta):
     def azul_client(self):
         return AzulClient()
 
+    def setUp(self) -> None:
+        super().setUp()
+        self.pruning_seed = random.randint(0, sys.maxsize)
```

Comment:
Should probably log the seed here now so we don't have to log it everywhere it is used.

Reply:
```suggestion
        self.random_seed = random.randint(0, sys.maxsize)
```